docs/page-objects/items.rst

```diff
@@ -42,6 +42,13 @@ To keep your code maintainable, we recommend you to:
     :ref:`page object classes <page-object-classes>`, e.g. using :ref:`field
     processors <field-processors>`.
 
+    Having code that makes item field values different from their counterpart
+    page object field values can subvert the expectations of users of your
+    code, which might need to access page object fields directly, for example
+    for `field subset selection`_.
+
+    .. _field subset selection: https://github.com/scrapinghub/web-poet/issues/115
```

Comment:
Still, I'd avoid linking from the docs to open issues. We don't need any additional features to be implemented in web-poet to justify this suggestion, as users can already use individual fields of the page objects.